### ISSUE TITLE
bad indentation error when rendering

### ISSUE THREAD

**Issue by wafflesatdawn** [quarto-dev/quarto-cli#5476]
### Bug description

I recently reinstalled quarto and now the render function is no longer working. When I use the CLI or the render button I get `YAMLError: bad indentation of a mapping entry at line 5, column 10:
      jupyter: python3`

I've rendered these same files before, using the current indents (or lack thereof), and even when I try various combinations of single, double indent or spaces, the same error appears. I'm not sure what's gone wrong.

Trying to attach a .qmd says that the file is not supported [index.txt](https://github.com/quarto-dev/quarto-cli/files/11436666/index.txt)

I'm using Windows 10, VS Code
```
Checking versions of quarto binary dependencies...
      Pandoc version 3.1.1: OK
      Dart Sass version 1.55.0: OK
[>] Checking versions of quarto dependencies......OK
[>] Checking Quarto installation......OK
      Version: 1.3.319
      Path: C:\Users\Welp Windows 10\AppData\Local\Programs\Quarto\bin
      CodePage: 1252

[>] Checking basic markdown render....OK

[>] Checking Python 3 installation....OK
      Version: 3.11.1
      Path: C:/Users/Welp Windows 10/AppData/Local/Programs/Python/Python311/python.exe
      Jupyter: 5.1.3
      Kernels: python3

(\) Checking Jupyter engine render....0.00s - Debugger warning: It seems that frozen modules are being used, which may
0.00s - make the debugger miss breakpoints. Please pass -Xfrozen_modules=off
0.00s - to python to disable frozen modules.
0.00s - Note: Debugging will proceed. Set PYDEVD_DISABLE_FILE_VALIDATION=1 to disable this validation.
0.00s - Debugger warning: It seems that frozen modules are being used, which may
0.00s - make the debugger miss breakpoints. Please pass -Xfrozen_modules=off
0.00s - to python to disable frozen modules.
0.00s - Note: Debugging will proceed. Set PYDEVD_DISABLE_FILE_VALIDATION=1 to disable this validation.
[>] Checking Jupyter engine render....OK

[>] Checking R installation...........(None)
```

### Checklist

- [X] Please include a minimal, fully reproducible example in a single .qmd file? Please provide the whole file rather than the snippet you believe is causing the issue.
- [X] Please [format your issue](https://quarto.org/bug-reports.html#formatting-make-githubs-markdown-work-for-us) so it is easier for us to read the bug report.
- [ ] Please document the RStudio IDE version you're running (if applicable), by providing the value displayed in the "About RStudio" main menu dialog?
- [x] Please document the operating system you're running. If on Linux, please provide the specific distribution.
- [X] Please provide the output of `quarto check` so we know which version of quarto and its dependencies you're running.

**Comment by cscheid** [comment]
Can you install the latest version of quarto? You're running 1.3.319, and the latest version is 1.3.340. I can't actually reproduce that in 1.3.340:

```
$ quarto render issue-5476.qmd

Executing 'issue-5476.ipynb'
  Cell 1/2...Done
  Cell 2/2...Done

pandoc
  to: >-
    markdown_strict+raw_html+all_symbols_escapable+backtick_code_blocks+fenced_code_blocks+space_in_atx_header+intraword_underscores+lists_without_preceding_blankline+shortcut_reference_links+autolink_bare_uris+emoji+footnotes+gfm_auto_identifiers+pipe_tables+strikeout+task_lists+tex_math_dollars+yaml_metadata_block+definition_lists+smart
  output-file: issue-5476.md
  standalone: true
  default-image-extension: png
  wrap: preserve

metadata
  title: Quarto Basics
  date: '2023-04-21'
  draft: true
  jupyter: python3

Output created: issue-5476.md

$ quarto check

[✓] Checking versions of quarto binary dependencies...
      Pandoc version 3.1.1: OK
      Dart Sass version 1.55.0: OK
[✓] Checking versions of quarto dependencies......OK
[✓] Checking Quarto installation......OK
      Version: 1.3.340
      Path: /Users/cscheid/repos/github/cscheid/quarto-regress/releases/v1.3.340/bin
```

**Comment by wafflesatdawn** [comment]
I just updated and still get the same error.

**Comment by cscheid** [comment]
I'm out of ideas. The change must be from some difference in our environments. I can only suspect that some library upgrade changed your YAML parser. This is the output of my `pip freeze`, which should describe the python environment I'm running:

```
$ pip freeze
aiofiles==22.1.0
aiosqlite==0.18.0
altair==4.2.0
anyio==3.6.2
appnope==0.1.3
argon2-cffi==21.3.0
argon2-cffi-bindings==21.2.0
arrow==1.2.3
asttokens==2.2.1
astunparse==1.6.3
async-generator==1.10
attrs==22.2.0
Babel==2.11.0
backcall==0.2.0
beautifulsoup4==4.11.1
binaryornot==0.4.4
black==23.1.0
bleach==5.0.1
bokeh==2.4.3
branca==0.6.0
certifi==2022.12.7
cffi==1.15.1
chardet==5.1.0
charset-normalizer==3.0.1
click==8.1.3
click-plugins==1.1.1
cligj==0.7.2
colorcet==3.0.1
comm==0.1.2
contourpy==1.0.7
cookiecutter==2.1.1
coverage==7.0.5
cycler==0.11.0
Cython==0.29.33
dateparser==1.1.8
debugpy==1.6.5
decorator==5.1.1
defusedxml==0.7.1
descartes==1.1.0
entrypoints==0.4
execnb==0.1.4
executing==1.2.0
fastcore==1.5.28
fastjsonschema==2.16.2
Fiona==1.8.22
flake8==6.0.0
fonttools==4.38.0
fqdn==1.5.1
geopandas==0.12.2
ghapi==1.0.3
holoviews==1.15.4
httplib2==0.21.0
humanize==4.6.0
hvplot==0.8.2
idna==3.4
importlib-metadata==6.0.0
ipyflex==0.2.6
ipykernel==6.20.2
ipyleaflet==0.17.2
ipython==8.8.0
ipython-genutils==0.2.0
ipywidgets==8.0.4
isoduration==20.11.0
jedi==0.18.2
Jinja2==3.1.2
jinja2-time==0.2.0
joblib==1.2.0
json5==0.9.11
jsonpointer==2.3
jsonschema==4.17.3
jupyter==1.0.0
jupyter-c-kernel==1.2.2
jupyter-cache==0.5.0
jupyter-console==6.4.4
jupyter-events==0.6.3
jupyter-ydoc==0.2.3
jupyter_client==7.4.9
jupyter_core==5.1.3
-e git+ssh://git@github.com/quarto-dev/quarto@10eec48bcefbc987a67bf9ccd53141673777ef6a#egg=jupyter_quarto&subdirectory=apps/jupyterlab
jupyter_server==2.1.0
jupyter_server_fileid==0.8.0
jupyter_server_terminals==0.4.4
jupyter_server_ydoc==0.8.0
jupyterlab==3.6.2
jupyterlab-pygments==0.2.2
jupyterlab-widgets==3.0.5
jupyterlab_server==2.19.0
kiwisolver==1.4.4
loguru==0.6.0
lxml==4.9.2
Markdown==3.4.1
MarkupSafe==2.1.2
mathjax==0.1.2
matplotlib==3.6.3
matplotlib-inline==0.1.6
maya==0.6.1
mccabe==0.7.0
mistune==2.0.4
mizani==0.8.1
mock==5.0.1
munch==2.5.0
mypy-extensions==1.0.0
nbclassic==0.4.8
nbclient==0.5.13
nbconvert==7.2.8
nbdev-numpy==0.0.586
nbdev-stdlib==0.0.603
nbformat==5.7.3
nest-asyncio==1.5.6
nose==1.3.7
notebook==6.5.2
notebook_shim==0.2.2
numpy==1.24.1
packaging==23.0
palettable==3.3.0
pandas==2.0.0
pandocfilters==1.5.0
panel==0.14.3
panflute==2.3.0
param==1.12.3
parso==0.8.3
pathspec==0.11.0
patsy==0.5.3
pendulum==2.1.2
pexpect==4.8.0
pickleshare==0.7.5
Pillow==9.4.0
pip-conflict-checker==0.6.0
pip-review==1.3.0
platformdirs==2.6.2
plotly==5.13.1
plotly-express==0.4.1
plotnine==0.10.1
polars==0.17.12
poyo==0.5.0
prometheus-client==0.15.0
prompt-toolkit==3.0.36
psutil==5.9.4
ptyprocess==0.7.0
pure-eval==0.2.2
pyarrow==11.0.0
pycodestyle==2.10.0
pycparser==2.21
pyct==0.5.0
pyflakes==3.0.1
Pygments==2.14.0
pyparsing==3.0.9
pyproj==3.4.1
pyrsistent==0.19.3
python-dateutil==2.8.2
python-json-logger==2.0.4
python-slugify==8.0.1
pytz==2022.7.1
pytz-deprecation-shim==0.1.0.post0
pytzdata==2020.1
pyviz-comms==2.2.1
PyYAML==6.0
pyzmq==25.0.0
qtconsole==5.4.0
QtPy==2.3.0
quarto==0.1.0
regex==2023.3.23
requests==2.28.2
requests-unixsocket==0.3.0
rfc3339-validator==0.1.4
rfc3986-validator==0.1.1
scikit-learn==1.2.0
scipy==1.10.0
seaborn==0.12.2
Send2Trash==1.8.0
shapely==2.0.0
six==1.16.0
sklearn==0.0.post1
snaptime==0.2.4
sniffio==1.3.0
soupsieve==2.3.2.post1
SQLAlchemy==1.4.46
stack-data==0.6.2
statsmodels==0.13.5
stravaio==0.0.9
svg.py==1.4.1
tabulate==0.9.0
tenacity==8.1.0
terminado==0.17.1
testpath==0.6.0
text-unidecode==1.3
threadpoolctl==3.1.0
tinycss2==1.2.1
tomli==2.0.1
toolz==0.12.0
tornado==6.2
tqdm==4.64.1
traitlets==5.8.1
traittypes==0.2.1
typing_extensions==4.4.0
tzdata==2023.2
tzlocal==4.3
uri-template==1.2.0
urllib3==1.26.14
vega-datasets==0.9.0
watchdog==2.2.1
wcwidth==0.2.6
webcolors==1.12
webencodings==0.5.1
websocket-client==1.4.2
widgetsnbextension==4.0.5
xyzservices==2022.9.0
y-py==0.5.9
ypy-websocket==0.8.2
zipp==3.11.0
```

Can you provide yours?

**Comment by wafflesatdawn** [comment]
Sure:
```aiofiles==22.1.0
aiosqlite==0.19.0
anyio==3.6.2
appdirs==1.4.4
argon2-cffi==21.3.0
argon2-cffi-bindings==21.2.0
arrow==1.2.3
astroid==2.12.13
asttokens==2.2.1
async-generator==1.10
attrs==22.2.0
Babel==2.12.1
backcall==0.2.0
Backtesting==0.3.3
beautifulsoup4==4.11.1
black==23.1.0
bleach==6.0.0
blis==0.7.9
bokeh==2.4.2
catalogue==2.0.8
cattrs==22.2.0
certifi==2022.12.7
cffi==1.15.1
charset-normalizer==2.1.1
click==8.1.3
colorama==0.4.6
colour==0.1.5
comm==0.1.2
confection==0.0.4
contourpy==1.0.6
cryptography==38.0.4
cycler==0.11.0
cymem==2.0.7
debugpy==1.6.5
decorator==5.1.1
defusedxml==0.7.1
dill==0.3.6
dtreeviz==2.2.1
entrypoints==0.4
exchange-calendars==4.2.5
executing==1.2.0
fastai==2.7.12
fastcore==1.5.29
fastdownload==0.0.7
fastjsonschema==2.16.3
fastprogress==1.0.3
filelock==3.10.7
fonttools==4.38.0
fqdn==1.5.1
frozendict==2.3.4
graphviz==0.20.1
greenlet==2.0.1
h11==0.14.0
html5lib==1.1
idna==3.4
iniconfig==2.0.0
ipykernel==6.20.2
ipython==8.8.0
ipython-genutils==0.2.0
isoduration==20.11.0
isort==5.11.3
jedi==0.18.2
Jinja2==3.1.2
joblib==1.2.0
json5==0.9.11
jsonpointer==2.3
jsonschema==4.17.3
jupyter-events==0.6.3
jupyter-ydoc==0.2.4
jupyter_client==7.4.9
jupyter_core==5.1.3
jupyter_server==2.5.0
jupyter_server_fileid==0.9.0
jupyter_server_terminals==0.4.4
jupyter_server_ydoc==0.8.0
jupyterlab==3.6.3
jupyterlab-pygments==0.2.2
jupyterlab_server==2.22.1
kaggle==1.5.13
kiwisolver==1.4.4
korean-lunar-calendar==0.3.1
langcodes==3.3.0
lazy-object-proxy==1.8.0
lxml==4.9.2
MarkupSafe==2.1.1
matplotlib==3.6.2
matplotlib-inline==0.1.6
mccabe==0.7.0
mistune==2.0.5
mpmath==1.3.0
msgpack==1.0.4
multitasking==0.0.11
murmurhash==1.0.9
mypy-extensions==0.4.3
nbclassic==1.0.0
nbclient==0.7.3
nbconvert==7.3.1
nbformat==5.8.0
nest-asyncio==1.5.6
networkx==3.0
notebook==6.5.4
notebook_shim==0.2.3
numpy==1.24.2
outcome==1.2.0
packaging==22.0
pandas==1.5.2
pandas-datareader==0.10.0
pandocfilters==1.5.0
parso==0.8.3
pathspec==0.10.3
pathy==0.10.1
patsy==0.5.3
pickleshare==0.7.5
Pillow==9.3.0
platformdirs==2.6.0
pluggy==1.0.0
preshed==3.0.8
prometheus-client==0.16.0
prompt-toolkit==3.0.36
psutil==5.9.4
pure-eval==0.2.2
pycparser==2.21
pydantic==1.10.4
Pygments==2.14.0
pylint==2.15.9
pyluach==2.0.2
PyMySQL==1.0.2
pyparsing==3.0.9
pyrate-limiter==2.8.5
pyrsistent==0.19.3
PySocks==1.7.1
pytest==7.2.2
python-dateutil==2.8.2
python-json-logger==2.0.7
python-slugify==8.0.1
pytz==2022.7
pywin32==305
pywinpty==2.0.10
PyYAML==6.0
pyzmq==25.0.0
requests==2.28.1
requests-cache==0.9.8
requests-ratelimiter==0.4.0
rfc3339-validator==0.1.4
rfc3986-validator==0.1.1
scikit-learn==1.2.0
scipy==1.9.3
seaborn==0.12.2
sec-cik-mapper==2.1.0
sec-edgar-api==1.0.0
selenium==4.8.2
Send2Trash==1.8.2
sentencepiece==0.1.98
simplegeneric==0.8.1
six==1.16.0
smart-open==6.3.0
sniffio==1.3.0
sortedcontainers==2.4.0
soupsieve==2.3.2.post1
spacy==3.5.1
spacy-legacy==3.0.12
spacy-loggers==1.0.4
SQLAlchemy==1.4.45
srsly==2.4.6
sseclient-py==1.7.2
stack-data==0.6.2
statsmodels==0.13.5
sympy==1.11.1
TA-Lib==0.4.25
terminado==0.17.1
text-unidecode==1.3
thinc==8.1.9
threadpoolctl==3.1.0
tinycss2==1.2.1
tomlkit==0.11.6
toolz==0.12.0
torch==2.0.0
torchvision==0.15.1
tornado==6.2
tqdm==4.65.0
traitlets==5.8.1
trio==0.22.0
trio-websocket==0.9.2
typer==0.7.0
typing_extensions==4.4.0
uri-template==1.2.0
url-normalize==1.4.3
urllib3==1.26.13
wasabi==1.1.1
wcwidth==0.2.6
webcolors==1.13
webencodings==0.5.1
websocket-client==1.5.1
websockets==10.4
wrapt==1.14.1
wsproto==1.2.0
xyzservices==2022.9.0
y-py==0.5.9
yfinance==0.2.12
yfinance-cache==0.3.5
ypy-websocket==0.8.2```

**Comment by cscheid** [comment]
PyYAML is at the same version. I'm totally stumped now...

**Comment by jjallaire** [comment]
Our Python YAML parsing is not an external dependency (we have a vendored library for that)

**Comment by mcanouil** [comment]
@wafflesatdawn Do you have a `_quarto.yml` file alongside your Quarto document? 🤔

**Comment by wafflesatdawn** [comment]
Yes, here it is
[_quarto.yml.txt](https://github.com/quarto-dev/quarto-cli/files/11443503/_quarto.yml.txt)

I was looking at the file history before uploading and noticed I had a `:` after `hugo-md` in some versions so I tried changing it both ways but that doesn't seem to have made a difference.

**Comment by cscheid** [comment]
```
format:
  hugo-md
    code-fold: false
```

You're missing the `:` there, that's the bug.

**Comment by mcanouil** [comment]
@wafflesatdawn Could you share (if possible) your project as a Git repository?

**Comment by wafflesatdawn** [comment]
I've just added both of you as collaborators on the repo so let me know if you can access it

**Comment by cscheid** [comment]
I can see you repo, and there's definitely something bad happening here. I'll get back to you.

**Comment by cscheid** [comment]
Can you provide the _entire_ error message, from invocation to output?

Your repository renders without problems for me once I removed that bad _quarto.yml from a parent directory. The _quarto.yml file doesn't need to be on your repository. It might be in one of the _parent directories_.

**Comment by wafflesatdawn** [comment]
Sure thing

```
PS C:\Users\Welp Windows 10\Documents\10 - 19 career\13 persona\professional> quarto preview "c:/Users/Welp Windows 10/Documents/10 - 19 career/13 persona/professional/content/posts/helloworld/index.qmd" --no-browser --no-watch-inputs
ERROR: YAMLError: bad indentation of a mapping entry at line 5, column 10:
      jupyter: python3
             ^

YAMLError: bad indentation of a mapping entry at line 5, column 10:
      jupyter: python3
             ^
    at generateError (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:10719:12)       
    at throwError (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:10722:11)
    at readBlockMapping (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:11361:20)    
    at composeNode (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:11528:84)
    at readDocument (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:11642:5)
    at loadDocuments (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:11679:9)        
    at load (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:11684:23)
    at parse3 (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:11694:12)
    at parseWithNiceErrors (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:19862:16) 
    at readYamlFromMarkdown (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:19793:17)
```

**Comment by cscheid** [comment]
Thanks for the patience. I don't know enough PowerShell to tell you how to do this exactly, but can you add `QUARTO_DENO_EXTRA_OPTIONS=--v8-flags=--stack-trace-limit=100` to your environment and rerun that command? We'll get a longer stack trace and it'll be more likely to show useful information.

**Comment by wafflesatdawn** [comment]
That command didn't work in Powershell and `quarto help` made no mentions of extra options so I started a bash instead. It accepted the command but I'm not sure if I did it correctly because there was no additional output. After that I ran `quarto render` again, both with and without `_quarto.yml` in the config folder.

Maybe there's some arcane shortcut I triggered somewhere that's telling quarto to parse weirdly?

```

Welp Windows 10@SAMSARA MINGW64 ~/Documents/10 - 19 career/13 persona/professional (master)
$ QUARTO_DENO_EXTRA_OPTIONS=--v8-flags=--stack-trace-limit=100

Welp Windows 10@SAMSARA MINGW64 ~/Documents/10 - 19 career/13 persona/professional (master)
$ quarto render ./content/posts/katex.qmd
ERROR: YAMLError: bad indentation of a mapping entry at line 5, column 10:
      jupyter: python3
             ^

YAMLError: bad indentation of a mapping entry at line 5, column 10:
      jupyter: python3
             ^
    at generateError (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:10719:12)
    at throwError (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:10722:11)
    at readBlockMapping (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:11361:20)
    at composeNode (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:11528:84)
    at readDocument (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:11642:5)
    at loadDocuments (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:11679:9)
    at load (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:11684:23)
    at parse3 (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:11694:12)
    at parseWithNiceErrors (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:19862:16)
    at readYamlFromMarkdown (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:19793:17)
```

**Comment by cscheid** [comment]
If you're going to use bash, then you need to say, instead,

```
$ QUARTO_DENO_EXTRA_OPTIONS=--v8-flags=--stack-trace-limit=100 quarto render ./content/posts/katex.qmd
```

or first `export` the variable and then call quarto render:

```
$ export QUARTO_DENO_EXTRA_OPTIONS=--v8-flags=--stack-trace-limit=100 
$ quarto render ./content/posts/katex.qmd
```

**Comment by cscheid** [comment]
I notice now that the file that's rendering for you is missing in the repository. I don't see a `katex.qmd` file here: https://github.com/wafflesatdawn/professional/tree/master/content/posts

**Comment by wafflesatdawn** [comment]
Oh, let me push changes to Github. (It's taking an unusual amount of time for some reason.)

Meanwhile here's the combined command

```

Welp Windows 10@SAMSARA MINGW64 ~/Documents/10 - 19 career/13 persona/professional (master)
$  QUARTO_DENO_EXTRA_OPTIONS=--v8-flags=--stack-trace-limit=100 quarto render ./content/posts/helloworld/index.qmd
ERROR: YAMLError: bad indentation of a mapping entry at line 5, column 10:
      jupyter: python3
             ^

YAMLError: bad indentation of a mapping entry at line 5, column 10:
      jupyter: python3
             ^
    at generateError (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:10719:12)
    at throwError (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:10722:11)
    at readBlockMapping (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:11361:20)
    at composeNode (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:11528:84)
    at readDocument (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:11642:5)
    at loadDocuments (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:11679:9)
    at load (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:11684:23)
    at parse3 (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:11694:12)
    at parseWithNiceErrors (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:19862:16)
    at readYamlFromMarkdown (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:19793:17)

```

**Comment by cscheid** [comment]
No luck here, that file continues to render without a problem. Can you add @cderv to your repository? He's our Windows expert and might be able to help. I'm out of ideas.

**Comment by wafflesatdawn** [comment]
Will do. I have to clean up my git commits first though; it seems that I carelessly triggered the filesize limit and need to remove references to the large file after deleting it. Then I'll push my changes and maybe that'll uncover the issues here.

**Comment by wafflesatdawn** [comment]
Ok, the sync was successful. I've added @cderv as requested.

**Comment by cderv** [comment]
Thanks @wafflesatdawn . 

I cloned you repo master branch, and I can't reproduce the issue. Do you have other un-commited file or unfollowed file that could impact what happens for you ? 

> I don't know enough PowerShell to tell you how to do this exactly, but can you add QUARTO_DENO_EXTRA_OPTIONS=--v8-flags=--stack-trace-limit=100 

Regarding environment variable in powershell , you can do in your terminal
```powershell
$env:QUARTO_DENO_EXTRA_OPTIONS="--v8-flags=--stack-trace-limit=100"
````
and then run quarto

**Comment by wafflesatdawn** [comment]
Thanks for taking a look! I've got the following in my gitignore
```
/.quarto/
public
*.csv
*.clj
```
The public folder is just what Hugo renders, I think quarto put itself as the first item, and the .clj is unrelated. I've just written a few more paragraphs in a .qmd file since my last commit/sync, but I don't think those meaningfully affected anything--the unmodified files won't render either.

Oh, I guess I should mention I updated Deno to the latest version. While doing `quarto convert` I saw the message and thought it might help, but no luck. 

Powershell:

```
PS C:\Users\Welp Windows 10\Documents\10 - 19 career\13 persona\professional> $env:QUARTO_DENO_EXTRA_OPTIONS="--v8-flags=--stack-trace-limit=100"
PS C:\Users\Welp Windows 10\Documents\10 - 19 career\13 persona\professional> quarto render ./content/posts/blueberry-yield/index.qmd
ERROR: YAMLError: bad indentation of a mapping entry at line 5, column 10:
      jupyter: python3
             ^

YAMLError: bad indentation of a mapping entry at line 5, column 10:
      jupyter: python3
             ^
    at generateError (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:10719:12)
    at throwError (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:10722:11)
    at readBlockMapping (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:11361:20)
    at composeNode (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:11528:84)
    at readDocument (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:11642:5)
    at loadDocuments (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:11679:9)
    at load (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:11684:23)
    at parse3 (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:11694:12)
    at parseWithNiceErrors (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:19862:16)
    at readYamlFromMarkdown (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:19793:17)
```
```
PS C:\Users\Welp Windows 10\Documents\10 - 19 career\13 persona\professional> quarto render ./content/posts/katex.qmd
ERROR: YAMLError: bad indentation of a mapping entry at line 5, column 10:
      jupyter: python3
             ^

YAMLError: bad indentation of a mapping entry at line 5, column 10:
      jupyter: python3
             ^
    at generateError (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:10719:12)
    at throwError (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:10722:11)
    at readBlockMapping (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:11361:20)
    at composeNode (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:11528:84)
    at readDocument (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:11642:5)
    at loadDocuments (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:11679:9)
    at load (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:11684:23)
    at parse3 (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:11694:12)
    at parseWithNiceErrors (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:19862:16)
    at readYamlFromMarkdown (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:19793:17)
```
```
PS C:\Users\Welp Windows 10\Documents\10 - 19 career\13 persona\professional> quarto render ./content/posts/helloworld/index.qmd
ERROR: YAMLError: bad indentation of a mapping entry at line 5, column 10:
      jupyter: python3
             ^

YAMLError: bad indentation of a mapping entry at line 5, column 10:
      jupyter: python3
             ^
    at generateError (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:10719:12)
    at throwError (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:10722:11)
    at readBlockMapping (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:11361:20)
    at composeNode (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:11528:84)
    at readDocument (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:11642:5)
    at loadDocuments (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:11679:9)
    at load (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:11684:23)
    at parse3 (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:11694:12)
    at parseWithNiceErrors (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:19862:16)
    at readYamlFromMarkdown (file:///C:/Users/Welp%20Windows%2010/AppData/Local/Programs/Quarto/bin/quarto.js:19793:17)
```

**Comment by cderv** [comment]
That is really a puzzling issue, especially as we can't reproduce from your own source code project. 

For test purpose, can you cloning you repo on another computer or just another place in your current computer (under Documents/ maybe) and see if you can reproduce your error ? 

I wonder if it comes from your whole environment, or the context of you current project.

**Comment by wafflesatdawn** [comment]
Well, I cloned into another directory, tested and it seemed to work, then I renamed the original directory and tried again, with results seeming to work.

```
PS C:\Users\Welp Windows 10\Documents\10 - 19 career\13 persona> git clone https://github.com/wafflesatdawn/professional
Cloning into 'professional'...
remote: Enumerating objects: 247, done.
remote: Counting objects: 100% (247/247), done.
remote: Compressing objects: 100% (163/163), done.
Receiving objects:  98% (243/247), 5.28 MiB | 10.54 MiB/s-reused 0
Receiving objects: 100% (247/247), 10.37 MiB | 11.53 MiB/s, done.
Resolving deltas: 100% (61/61), done.
PS C:\Users\Welp Windows 10\Documents\10 - 19 career\13 persona> cd .\professional\
PS C:\Users\Welp Windows 10\Documents\10 - 19 career\13 persona\professional> quarto render ./content/posts/helloworld/index.qmd

Starting python3 kernel...Done

Executing 'index.ipynb'
  Cell 1/2...Done
  Cell 2/2...Done

pandoc 
  to: >-
    markdown_strict+raw_html+all_symbols_escapable+backtick_code_blocks+fenced_code_blocks+space_in_atx_header+intraword_underscores+lists_without_preceding_blankline+shortcut_reference_links+autolink_bare_uris+emoji+footnotes+gfm_auto_identifiers+pipe_tables+strikeout+task_lists+tex_math_dollars+yaml_metadata_block+definition_lists+smart
  output-file: index.md
  standalone: true
  default-image-extension: png
  wrap: preserve

metadata
  title: Quarto Basics
  date: '2023-04-21'
  draft: true
  jupyter: python3

Output created: index.md
```

This is the most nondeterministic behavior I've seen in a while. I'm going to continue rendering the rest of my files, for which I need to copy some files over, but maybe they'll go smoothly too. Will report back shortly

**Comment by cscheid** [comment]
I still think that, in your old directory that you renamed away, you have a `_quarto.yml` file that is being picked up by accident by quarto.

**Comment by wafflesatdawn** [comment]
The rest of my .qmd files rendered perfectly in the new directory. I'm still surprised this ended up being the solution. One thing that comes to mind now is that I had used `freeze: true` at one point before removing it and deleting everything I could see that was related to it. Other than that I have no idea. And I searched again for a `_quarto.yml` and found nothing.

Thanks for your help everyone!
